src/Approximations/overapproximate.jl

```diff
@@ -1636,6 +1647,31 @@ function overapproximate(X::LazySet{N}, ::Type{<:Zonotope},
     return Zonotope(c, G)
 end
 
+function _overapproximate_zonotope_cpa(X::AbstractPolytope, ::Type{<:Zonotope},
+                                       dirs=OctDirections)
+    n = dim(X)
+    p = Vector(undef, Int(floor(n/2)))
+    j = 1
+    if isa(X, VPolytope)
```

Comment:
why not dispatch on `X`? you can use a helper function to reduce the repeated code.